R/plots.R

```diff
@@ -491,6 +490,31 @@ plot_sample_heatmap <- function(obj,
   p
 }
 
+#' Plot volcano plot
+plot_volcano = function(obj, which_beta, which_model = 'full',
+    sig_level = 0.10,
+    point_alpha = 0.2,
+    sig_color = 'red',
+    highlight = NULL
+    ) {
+    stopifnot( is(obj, 'sleuth') )
+    
+    res <- sleuth_results(obj, which_beta, which_model, rename_cols = FALSE,
+        show_all = FALSE)
+    res <- dplyr::mutate(res, significant = qval < sig_level)
+    
+    suppressWarnings({
```

Comment:
let's not suppressWarnings here. if there are a lot of points missing, the user should know